ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -185,69 +224,116 @@ const POPULATIONS = [
   {
     field: 'gnomad_mito',
     fieldTitle: 'gnomAD mito',
+    titleContainer: sectionTitle,
+    section: HOM_SECTION,
     precision: 3,
-    urls: { [GENOME_VERSION_38]: 'gnomad.broadinstitute.org' },
-    queryParams: { [GENOME_VERSION_38]: 'dataset=gnomad_r3' },
+    ...GNOMAD_URL_INFO,
+  },
+  {
+    field: 'gnomad_mito_heteroplasmy',
+    fieldTitle: 'gnomAD mito',
+    titleContainer: sectionTitle,
+    section: HET_SECTION,
+    precision: 3,
+    ...GNOMAD_URL_INFO,
   },
   {
     field: 'helix',
     fieldTitle: 'Helix mito',
+    titleContainer: sectionTitle,
+    section: HOM_SECTION,
     precision: 3,
   },
+  {
+    field: 'helix_heteroplasmy',
+    fieldTitle: 'Helix mito',
+    titleContainer: sectionTitle,
+    section: HET_SECTION,
+    precision: 3,
+  },
+]
+
+const acDisplay = (populations, popConfig) => ([
+  `${popConfig.fieldTitle}: ${populations[popConfig.field].ac} out of ${populations[popConfig.field].an}`,
+])
+const DETAIL_SECTIONS = [
+  {
+    name: GLOBAL_AF_SECTION,
+    hasDetail: (populations, popConfig) => populations[popConfig.field] && populations[popConfig.field].filter_af &&
```

Comment:
the only part of `populations` and `popConfig` in any of the `hasDetail` functions is `populations[popConfig.field]` it would be much more readable/ maintainable if you called it with `hasDetail(populations[popConfig.field])`